### PR TITLE
DDT Reference Manual, and LISP Machine Progress Report

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
With permission from @ericosman.

### CODE REVIEW

**Review by eswenson1 — APPROVED** [review]

**Comment by larsbrinkhoff** [comment]
I'm adding another commit for *LISP Machine Progress Report*, typeset with the new TJ6.

**Comment by eswenson1** [comment]
ok.  Do you want me to reapprove once you commit, or not bother?

**Comment by larsbrinkhoff** [comment]
It sounds like you don't disapprove, so no need to approve again in that case.

**Comment by larsbrinkhoff** [comment]
Let me rephrase that.  Please comment or request changes if there's anything wrong.
